src/loaders/compute_tools/microtrait/microtrait.py

```diff
@@ -107,6 +198,11 @@ def _run_microtrait(tool_safe_data_id: str, data_id: str, fna_file: Path, genome
 
     trait_counts_df.to_csv(os.path.join(genome_dir, loader_common_names.TRAIT_COUNTS_FILE), index=False)
```

Comment:
Should this line not be deleted?

Reply:
I think we should still keep the original result from the tool. Just in case.

Reply:
Ok, I suppose it doesn't hurt

:+1: